purchase_ext_sst/views/purchase_order_views.xml

```diff
@@ -45,6 +45,9 @@
             <xpath expr="//notebook/page[2]//field[@name='date_planned']" position="attributes">
                 <attribute name="required">False</attribute>
             </xpath>
+            <xpath expr="//field[@name='date_order']" position="after">
+                <field name="supplier_update_lock" groups="base.group_system"/>
```

Comment:
`groups="base.group_system"` is used since staff accounts should not have the Administration Access

Reply:
Yes, this is as intended.